packages/blockchain-extension/package.json

```diff
@@ -1341,7 +1341,7 @@
         "pretest": "npm run updatePackageJSON",
         "test": "node ./build/test/runTests.js",
         "posttest": "remap-istanbul -i ./coverage/coverage.json -o ./coverage/coverage-remap.json && istanbul check-coverage ./coverage/coverage-remap.json",
-        "cucumber": "node ./build/cucumber/runCucumberTests.js",
+        "cucumber": "npm run updatePackageJSON && node ./build/cucumber/runCucumberTests.js",
```

Comment:
Why do we need to do this?

Reply:
Ah.. Tests failed in azure, and it looked like the changes I made modified activation events slightly. Maybe something to do with activating the new extension dependency? Talked with Caroline, who's had a similar problem, and the suggestion was to updatePackageJSON just before running the cucumber tests, to guarantee activation events happen in the correct order and therefore tests start with a clean slate. This is the solution used in V2.